### DISCUSSION TITLE
低軌道衛星通訊技術不斷突破，但「地表上」的管制才是最大難關？

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#142] [General]
之前有同學分享關於星鏈計畫的新聞，大部分都比較著重在它的應用層面，這邊提供另一個切角給大家參考：
低軌道衛星對於通訊服務的延展性有很大的幫助，但我們其實忽略了許多法規上的問題，因爲這些衛星其實都受控於不同國家的法律，文中有提到包括通訊頻率、電信監控的問題。
所以未來星鏈計畫的廣泛應用可能還需要多邊協商與合作才能最大化其效益。

參考來源：[低軌道衛星通訊技術不斷突破，但「地表上」的管制才是最大難關？](https://www.bnext.com.tw/article/65604/leo-satellite-manage)

**Comment by enoch-lightning** [comment]
確實，當一個科技發展出來，必須同時兼顧實用性與各國的法制規定，如此才能成功商業化。

**Comment by xCk27x** [comment]
之前系上的老師到南極做研究時，頻寬只有1.6Mb/s，而且是全工作站幾百到幾千人共用的，傳輸資料時實在很乏力。但幸好南極不隸屬於任何國家，星鏈計畫應該很快就能在南極產生作用。
第一階段的目標應該是快速建造涵蓋全球的衛星通訊網，在缺乏網路的的地方提供服務，目前能想到的應用包括救援，開發，科學研究。影響到的人數相對較少，在跟國家談判時應該也會比較簡單。

**Comment by MAKERRRofKEVIN** [comment]
之前有聽過訂定國內太空法的律師分享，這類國際規範的推展速度往往遠不及科技自身發展的速度，一方面來說這段無法可管的空窗期會造成混亂，但同時也能夠給那些洞察先機的拓荒者一個領先的機會

**Comment by syt-110** [comment]
之前在系上課程老師有提到頻寬的部分需要向上級單位申請，例如雷達、軍用設備......所用頻寬之於我國通訊中心(NCC)，所以在想是否低軌衛星已向上級申請
